### ISSUE TITLE
Regression: Resolver Issue in Mapping CRD with Ambassador version upgrade to 1.12.2

### ISSUE THREAD

**Issue by avinashkumar1289** [datawire/ambassador#3331]
We have recently upgraded the version of ambassador from 1.6.1 to 1.12.2.
However due to this all our Mapping CRD that uses resolver have stopped working
 

**To Reproduce**
1. Create a Mapping CRD with resolver (we need session affinity)
2. The endpoint does not work any more
3. I can also see the below warning in the Logs

```
level=warning msg="no endpoints for cluster cluster_https___tc_waconsole_9443_otls_a-0: k8s/avi/tc-waconsole/9443" func=github.com/datawire/ambassador/cmd/ambex.JoinEdsClusters file="github.com/datawire/ambassador/cmd/ambex/transforms.go:196" CMD=entrypoint PID=1 THREAD=/ambex
```
**Expected behavior**
The mapping should work .

**Versions (please complete the following information):**
 - Ambassador: 1.12.2
 - Kubernetes environment : Google Kubernetes Engine
 - Version : v1.17.15-gke.800

**Additional context**
These are my mappings
```
apiVersion: getambassador.io/v2
kind: Mapping
metadata:
  name: test-console
spec:
  prefix: /.*
  host: console\..*
  ambassador_id: test123
  case_sensitive: false
  host_regex: true
  prefix_regex: true
  rewrite: ""
  timeout_ms: 10000
  service: https://test-waconsole:9443
  resolver: endpoint
  load_balancer:
    policy: ring_hash
    cookie:
      name: console-affinity-cookie
      ttl: 200s
---
apiVersion: getambassador.io/v2
kind: KubernetesEndpointResolver
metadata:
  name: endpoint
spec:
  ambassador_id: test123 
---
```

**Comment by rhs** [comment]
A fix for this will be shipping soon. In the meantime you can workaround this issue by removing the "https://" from the `service:` field and instead specifying `tls: true`.

**Comment by khussey** [comment]
This is fixed in Ambassador 1.13.0, which is now available.
